modules/mcuboot/Kconfig

```diff
@@ -112,4 +112,13 @@ config ADD_MCUBOOT_MEDIATE_SIM_FLASH_DTS
 	  multi image update of the network core or to allow serial recovery of
 	  the network core.
 
+config MCUBOOT_USE_ALL_AVAILABLE_RAM
+	bool "Allow MCUBoot to use all available RAM (security implications)"
+	depends on TRUSTED_EXECUTION_NONSECURE || TRUSTED_EXECUTION_SECURE
+	default y if (BOARD = "thingy91_nrf9160") || BOARD_THINGY91_NRF9160
```

Comment:
Why are both of these conditions needed? Just use `default y if BOARD_THINGY91_NRF9160`

Reply:
I can do `BOARD_THINGY91_NRF9160 || BOARD_THINGY91_NRF9160_NS` or I could do `BOARD = "thingy91_nrf9160"`. I'll change it to use the Kconfigs instead of string matching.